<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMotorPage</class>
 <widget class="QWidget" name="VMotorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lstMotors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>711</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblMotorInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>891</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosition">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosition2">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeed">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>10</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSpeed">
   <property name="geometry">
    <rect>
     <x>1590</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Speed</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbRMove">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>60</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>RMove</string>
   </property>
   <widget class="QPushButton" name="btnRMoveM">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPitch3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRMoveP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPitch1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0.001</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPitch4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPitch5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btnPitch2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAMove">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>220</y>
     <width>481</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>AMove</string>
   </property>
   <widget class="QLineEdit" name="edtPos2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPos1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAMoveP1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Position1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAMoveP2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Position2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDelayTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Delay:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDelayTime">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRepeat12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat1-2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbStatus">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>370</y>
     <width>491</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnELP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>+EL</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnELM">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>-EL</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnORG">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>ORG</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnINP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>INP</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnALM">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>ALM</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMotion">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string>MOT</string>
    </property>
    <property name="icon">
     <iconset resource="Images.qrc">
      <normaloff>:/Images/Images/output.png</normaloff>
      <normalon>:/Images/Images/input.png</normalon>:/Images/Images/output.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnORG2">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>520</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimP">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>520</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Limit+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimM">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>520</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Limit-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>520</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnServoOn">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>590</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ServoOn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGoHome">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>660</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>GoHome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>660</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>590</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblError">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>590</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Error:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtError">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>620</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>660</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>590</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>891</width>
     <height>661</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Index</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <zorder>gbRMove</zorder>
  <zorder>lstMotors</zorder>
  <zorder>lblMotorInfo</zorder>
  <zorder>lblPosition</zorder>
  <zorder>lblPosition2</zorder>
  <zorder>lblSpeed</zorder>
  <zorder>btnSpeed</zorder>
  <zorder>gbAMove</zorder>
  <zorder>gbStatus</zorder>
  <zorder>btnORG2</zorder>
  <zorder>btnLimP</zorder>
  <zorder>btnLimM</zorder>
  <zorder>btnExit</zorder>
  <zorder>btnServoOn</zorder>
  <zorder>btnGoHome</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnStop</zorder>
  <zorder>lblError</zorder>
  <zorder>edtError</zorder>
  <zorder>btnReset</zorder>
  <zorder>btnLoad</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
